test/functional/page_objects/dashboard_page.ts

```diff
@@ -370,6 +383,7 @@ export class DashboardPageObject extends FtrService {
   }
 
   public async clickCreateDashboardPrompt() {
+    await this.forceSkipTour();
```

Comment:
Eventually, we should probably go through and clean up all the different ways the other test suites are accessing dashboard... This would really help to clean up all the places where I had to put `forceSkipTour()` to get CI to pass

Reply:
There has been some discussion (@timductive @kertal) around adding a `uiSetting` flag that would **way** clean up all of these tests - basically, there would be a flag that would disable all tours/notifications/callouts from appearing for users who are already comfortable with Kibana, and we could use this in our testing by defaulting it to `off` for all functional tests unless that specific feature is required.